index.html

```diff
@@ -17,10 +17,35 @@
   </head>
   <body>
     <header>
-      <div class="container"></div>
+      <div class="container" id="header">
+        <h1 class="column-full">Code Journal</h1>
+      </div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
+      <h1 class="column-full" id="new-entry">New Entry</h1>
+      <div data-view="entry-form">
+        <form class="row">
+          <div class="column-half">
+            <img
+              src="https://img.icons8.com/?size=100&id=qItA4cPLu8rh&format=png&color=c5c5c5"
+              alt="default placeholder image for journal entry"
+              class="img-placeholder" />
+          </div>
+          <div class="column-half">
+            <label for="title">Title</label><br />
+            <input id="title" type="text" name="title" required /><br />
+            <label for="url">Photo URL</label><br />
+            <input id="url" type="url" name="url" required />
```

Comment:
No `<br />` elements. If you need to add spacing between elements, you should be doing that in CSS!